test/statsCases/split-chunks-max-size/async/index.js

```diff
@@ -0,0 +1,7 @@
+Promise.all([
+  import(/* webpackChunkName: "async-b" */ "./a"),
+  import(/* webpackChunkName: "async-b" */ "./b")
+]).then(([a, b]) => {
+  a;
+  b;
+})
```

Comment:
Also include some modules into the initial chunk. Otherwise we don't see that the initial chunk isn't splitted.